public/app/features/explore/TraceView/components/DetailsPanel/DetailsPanel.tsx

```diff
@@ -0,0 +1,326 @@
+              </a>
+              SpanID: {span.spanID}
+            </small>
+          )}
+        </div>
+
+        <TabsBar>
+          {tabs.map((tab) => {
+            return (
+              <Tab
+                key={tab}
+                label={tab}
+                active={activeTab === tab}
+                counter={tabsCounters[tab]}
+                onChangeTab={() => setActiveTab(tab)}
+              />
+            );
+          })}
+        </TabsBar>
+
+        <TabContent className={styles.tab}>
+          {activeTab === TabLabels.Attributes && (
+            <div style={attrsTabStyle}>
+              <div className={styles.attributesContainer}>
+                <AccordianKeyValues
+                  className={styles.attributeValues}
+                  data={tags}
+                  label="Span Attributes"
+                  linksGetter={linksGetter}
+                  isOpen={true}
+                  interactive={false}
+                />
+              </div>
+              <div className={styles.attributesContainer}>
+                <AccordianKeyValues
+                  className={styles.attributeValues}
+                  data={process.tags}
+                  label="Resource Attributes"
+                  linksGetter={linksGetter}
+                  isOpen={true}
+                  interactive={false}
+                />
+              </div>
+            </div>
+          )}
+          {activeTab === TabLabels.Events && logs && logs.length > 0 && (
+            <AccordianLogs
+              linksGetter={linksGetter}
+              logs={logs}
+              openedItems={logsState.openedItems}
+              onItemToggle={(logItem) => detailLogItemToggle(span.spanID, logItem)}
+              timestamp={traceStartTime}
+            />
+          )}
+          {activeTab === TabLabels.Warnings && <TextList data={warnings} />}
+          {activeTab === TabLabels.StackTraces && <StackTraces stackTraces={stackTraces ?? []} />}
+          {activeTab === TabLabels.References && (
+            <AccordianReferences
+              data={references}
+              openedItems={referencesState.openedItems}
+              onItemToggle={(reference) => detailReferenceItemToggle(span.spanID, reference)}
+              createFocusSpanLink={createFocusSpanLink}
+            />
+          )}
+        </TabContent>
+      </ExploreDrawer>
+    </div>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  container: css({
+    label: 'DetailsPanel',
+    overflow: 'scroll',
+    position: 'sticky',
+    bottom: 0,
+    zIndex: `${theme.zIndex.activePanel}`,
+  }),
+  drawer: css`
+    margin: 0;
+    position: relative !important;
+  `,
+  header: css({
+    gap: '0 1rem',
+    padding: '0.6rem',
+    overflow: 'scroll',
+  }),
+  flexSpaceBetween: css({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  }),
+  tab: css`
```

Comment:
can we update this one to use an object, like this: 
```suggestion
  tab: css({
    label: 'DetailsPanelTab',
    padding: '0.5rem 1rem',
    '.json-markup': {
      lineHeight: '17px',
      fontSize: '13px',
      fontFamily: 'monospace',
      whiteSpace: 'pre-wrap',
    },
    '.json-markup-key': {
      fontWeight: 'bold',
    },
    '.json-markup-bool': {
      color: autoColor(theme, 'firebrick'),
    },
    '.json-markup-string': {
      color: autoColor(theme, 'teal'),
    },
    '.json-markup-null': {
      color: autoColor(theme, 'teal'),
    },
    '.json-markup-number': {
      color: autoColor(theme, 'blue', 'black'),
    },
  }),
```
I know that drawer can't be updated to use object because emotion doesn't support using `!important` in an object, but I thought we could at least update this one and reduce the number of betterer warnings